ci/test.sh

```diff
@@ -13,8 +13,9 @@ chown -R ubuntu:ubuntu /cache/
 echo '127.0.1.11 internal-stargate.local' >> /etc/hosts
 echo '127.0.1.12 internal-stargate.local' >> /etc/hosts
 
-# restart docker
-service docker --full-restart
+# Ensure we can connect to Docker daemon to resolve issue #1580, see:
+# https://dhananjay4058.medium.com/solving-docker-permission-denied-while-trying-to-connect-to-the-docker-daemon-socket-2e53cccffbaa
+sudo usermod -a -G docker ubuntu
```

Comment:
`sudo` is not required here. at this point in the script, you are running as root. line 20 is where it switches to `ubuntu`

Reply:
Ok. I thought I saw `sudo` earlier but it isn't actually used so my bad.